Client.xcodeproj/project.pbxproj

```diff
@@ -6604,14 +6604,14 @@
 				);
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "@executable_path/Frameworks";
-				MOZ_INTERNAL_URL_SCHEME = "fennec-aurora";
+				MOZ_INTERNAL_URL_SCHEME = "firefox-nightly";
 				MTL_ENABLE_DEBUG_INFO = NO;
 				OTHER_LDFLAGS = (
 					"-ObjC",
 					"-lxml2",
 				);
-				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_AURORA";
-				PRODUCT_BUNDLE_IDENTIFIER = "$(MOZ_BUNDLE_ID)";
+				OTHER_SWIFT_FLAGS = "-DMOZ_CHANNEL_NIGHTLY";
+				PRODUCT_BUNDLE_IDENTIFIER = "$(MOZ_BUNDLE_ID).enterprise";
```

Comment:
Instead of appending `.enterprise` here, you'll want to change the `MOZ_BUNDLE_ID` in the .xcconfig file so that the extensions get it as well. This will also allow the extensions to adhere to the rule that it needs to have the same prefixed bundle ID as the parent application since the extensions are setup to use `$(MOZ_BUNDLE_ID).$(PRODUCT_NAME:rfc1034identifier)`